edudoc/competitor-tutorial/tutorial.md

```diff
@@ -21,7 +22,9 @@ If you are not sure about something, always ask the Delegate!
 - **Runner**: Runners are responsible for moving puzzles between scramblers and judges.
 - **Scrambler**: Scramblers apply official scramble sequences on competitors' puzzles. They certify with their signature that the applied scramble was correct.
 
-## Objects at the competition {.page-break-before}
+<div class="page-break"></div>
```

Comment:
Why are you inserting the `<div>` manually now instead of the curly braces tag that was used before?

Reply:
I thought it would make a lot more sense to unify the page break styling of the two stylesheets. It doesn't make sense to have a separate page-break-before and a page-break-after. The page-break class is just page-break-before now, and the page-break-after class is gone. Also, I think it's a lot more intuitive for people to insert a line that visually shows where the page break is, instead of thinking that page-break means it'll break the page above. I think this is cleaner, although we could add another shortcut like {pb} that would be changed to the proper page-break div using sed during the build process. What do you think?

Reply:
No sed magic where you don't need it. sed makes sense for file paths because they are complicated and can also change at runtime. CSS classes cannot.

I am mostly opposed to manually inserting div tags when pandoc has a feature to do div styling for you. Having page-break-before and page-break-after gives you more fine-grained control about the layouts of your document; I believe this is a good thing. Same argument as above: The purpose of this PR is to update and unify the build logic. If you want to discuss page breaks, open a separate PR. But please keep them in curly braces inside this commit.

Reply:
@gregorbg okay, that's fair. Should I update it in edudoc-style then? Because the way it was done originally, it wasn't consistent between the two documents.

Reply:
I have no strong preference as long as you keep them in curly braces or pandoc ::::: div blocks.

Reply:
@gregorbg I will revert it to how it was originally then and update the edudoc stylesheet. Just fyi, I tried using the pandoc div blocks, but they must always take up at minimum two lines, which is why I replaced the page breaks with these ones everywhere. Looking back at it though, I do agree that it's still easier to type in the ::::: version.

Reply:
@gregorbg wait, I just realized that I got that the other way around. It was edudoc that had the two separate classes and style.css had one (page-break-before). What should I do then? Should I make edudoc-style.css consistent with style.css and update all uses of the classes to use ::::: instead of my manual html divs?

Reply:
I believe you should fulfill the following two criteria:
1. Offer both page-break-before and page-break-after for fine-grained user layout control.
2. Only insert manual `<div>` blocks where necessary. Use the pandoc Markdown extensions (i.e. ::::: and curly braces) wherever possible.

Reply:
Done